.github/actions/e2e/action.yml

```diff
@@ -0,0 +1,202 @@
+name: "e2e"
+description: "e2e"
+inputs:
+  e2e-test-id:
+    description: "Test id"
+    required: true
+  e2e-test-folder:
+    description: "Test folder"
+    required: true
+  e2e-test-provider:
+    description: "Test provider"
+    required: true
+  e2e-test-auth:
+    description: "Test auth"
+    required: true
+  output:
+    description: "Output path"
+    required: true
+  check-cs-env:
+    required: true
+    description: check cs env
+    default: "false"
+  check-ps-env:
+    required: true
+    description: check ps env
+    default: "false"
+  check-external-cs-env:
+    required: true
+    description: check external cs env
+    default: "false"
+  check-ps-cloud-env:
+    required: true
+    description: check ps cloud env
+    default: "false"
+  local-smart-runner-path: #
+    required: false
+    default: ".protractor-smartrunner"
+  e2e-tar-name: #
+    required: false
+    default: e2e.tar.gz
+  s3-build-path: #
+    required: true
+
+  apa-proxy-host:
+    required: true
+  e2e-proxy-host: #
+    required: true
+  apa-proxy: #
+    description: "proxy host"
+    required: true
+
+runs:
+  # description: try to download an e2e, if it was built previously
+  using: "composite"
+  steps:
+    - name: setup proxy env var
+      shell: bash
+      run: |
+        if [[ -z "${{ inputs.apa-proxy-host }}" ]]; then
+          echo "APA proxy host not found. terminating"
+          exit 1
+        fi
+        if [[ -z "${{ inputs.e2e-proxy-host }}" ]]; then
+          echo "e2e proxy host not found. terminating"
+          exit 1
+        fi
+
+        if [[ -z "${{ inputs.apa-proxy }}" ]]; then
+          echo "Custom proxy not set. using default "
+          echo "PROXY_HOST_BPM=${{ inputs.e2e-proxy-host }}" >> $GITHUB_ENV
```

Comment:
I think you are making everything more complicated on travis the flow was easy to understand 
PROXY_HOST_BPM was E2E_HOST as default
and only the e2e related to APA where doing the override 
"Process Cloud : People"
- PROXY_HOST_BPM=$E2E_HOST_APA